pkg/container/config.go

```diff
@@ -46,6 +46,7 @@ type Config struct {
 	PackageName           string
 	Mounts                []BindMount
 	Capabilities          Capabilities
+	CapAdd                []string // List of kernel capabilities to add to the container.
```

Comment:
Capability drop instead I think should be responsibility of the single runner (e.g. always dropping cap_sys_admin).

Reply:
Can we fold this into the Capabilities field above?

Reply:
Sure @jonjohnsonjr. Addressed with https://github.com/chainguard-dev/melange/pull/1702/commits/f2a5a53914168d5a0faae85173894d33113258a8